src/App.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import Calculator from './component/calculator';
+/* eslint-disable */
+
+class App extends Component {
+  render() {
+    return <Calculator />;
+  }
+}
+
```

Comment:
- [ ] Nice work so far 👍🏿 in your project. But as per project requirement, you are expected to use functional components instead of the class components you are currently using